woocommerce/woocommerce-android
# Add dev options screen UI

Closes: #7496 


### Description

This PR adds the ViewHolder and adapter for the RecyclerView and also the UI logic to the Fragment and ViewModel. 

Please note it also adds two types of list items  (toggleable, which is the one that will be implemented in this interaction, and non-toggleable for future interactions)

### Testing instructions


1. Open the app in debug
2. go to the Menu
3. go to the settings
4. Click on developer options
5. make sure there's 1 item showing (toggleable)

### Images/gif

<img width="350" alt="Screen Shot 2022-10-08 at 10 12 35 AM" src="https://user-images.githubusercontent.com/30724184/194711847-afa6b51c-5f2f-4aa5-b47f-d12a7f6edb44.png">
